package.json

```diff
@@ -90,7 +90,7 @@
     "@types/node": "^12.19.8",
     "@typescript-eslint/eslint-plugin": "~2.34.0",
     "@typescript-eslint/parser": "~2.34.0",
-    "cypress": "5.3.0",
+    "cypress": "6.6.0",
```

Comment:
Maybe this should be another task but just to mention it here that I think it is a good idea to upgrade prettier and eslint to the same version as user-office-frontend on both scheduler-frontend and scheduler-backend just to align the versions and rules everywhere.

Reply:
Yes, would be great but the last time I checked the react-calendar used some deprecated React features so upgrading the React version maybe not possible yet.